LDY/Day16/app/src/config/db.js

```diff
@@ -0,0 +1,12 @@
+const mysql = require("mysql");
```

Comment:
```suggestion
"use strict";

const mysql = require("mysql");
```
"use strict"; 선언을 추가하면 암묵적 전역 변수 생성이나 this 바인딩 오류를 방지할 수 있습니다!